pkg/config/orderer.go

```diff
@@ -284,3 +284,40 @@ func addOrdererPolicies(cg *cb.ConfigGroup, policyMap map[string]Policy, modPoli
 
 	return addPolicies(cg, policyMap, modPolicy)
 }
+
+// AddOrdererEndpoints adds an orderer's endpoint to an existing channel config transaction.
```

Comment:
`AddOrdererEndpoint` vs `AddOrdererEndpoints` - the Doc name doesn't match the function name.

Reply:
Sorry about it. 
I fixed the two nits in https://github.com/hyperledger/fabric/pull/891.